spec2vec/Spec2Vec.py

```diff
@@ -3,14 +3,44 @@
 
 
 class Spec2Vec:
+    """Calculate spec2vec similarity scores between a reference and a query.
 
+    Using a trained model, spectrum documents will be converted into spectrum
+    vectors. The spec2vec similarity is then the cosine similarity score between
+    two spectrum vectors.
+    """
```

Comment:
I added a small example. It is, however, not a fully testable/functional one because to get any meaningful scores it would be necessary to import a pre-trained model or a significant amount of data.

Reply:
Thats OK, we just should make sure it is working code in some way.